Gaming The Syllabus/Assets/Scripts/Combat/CombatActionButton.cs

```diff
@@ -7,9 +7,16 @@
 public class CombatActionButton : MonoBehaviour
 {
     [SerializeField] private PlayerAction playerAction;
+    [SerializeField] private GameObject abilityPage;
 
     public void OnClick()
     {
         CombatManager.PerformPlayerAction(playerAction);
     }
+
+    public void ShowAbilityPage()
+    {
+        abilityPage.SetActive(!abilityPage.activeSelf);
```

Comment:
I typically use activeInHierarchy here since that will also be false if a parent GameObject of the one you're checking is inactive (since that sets all the children inactive.) Not sure if it matters at all here but wanted to mention it